client/src/redux/prop-types.ts

```diff
@@ -47,9 +47,14 @@ export type MarkdownRemark = {
   };
 };
 
+export type MultipleChoiceAnswer = {
+  answer: string;
+  feedback: string;
```

Comment:
Is there a particular reason to not do this?

```suggestion
  feedback: string | null;
```

Reply:
✔️